### DISCUSSION TITLE
Cannot find package '@internal/pagesData'

### DISCUSSION THREAD

**Discussion by JeremieLitzler** [vuepress-theme-hope/vuepress-theme-hope#2343] [Q&A] [answered]
Hello dear fellow users,

I am currently using the beta 110 version and I would love to upgrade.

I am not using any customization in term of theme. Only what is available out of the box. Thanks to @Mister-Hope for all the features!

However, trying to upgrade to beta 111, I hit a road block:

```sh
✔ Initializing and preparing data - done in 11.22s
✔ Compiling with vite - done in 58.80s
✖ Rendering 172 pages - failed in 165ms
Error [ERR_MODULE_NOT_FOUND]: Cannot find package '@internal/pagesData' imported from D:\Git\GitHub\journal\node_modules\@vuepress\client\dist\chunk-X6HE2CCA.js
    at new NodeError (node:internal/errors:371:5)
    at packageResolve (node:internal/modules/esm/resolve:884:9)
    at moduleResolve (node:internal/modules/esm/resolve:929:18)
    at defaultResolve (node:internal/modules/esm/resolve:1044:11)
    at ESMLoader.resolve (node:internal/modules/esm/loader:422:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:222:40)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:76:40)
    at link (node:internal/modules/esm/module_job:75:36)
```

What am I missing?

I did delete the `node_modules` dir and year.lock file before running `yarn && yarn upgrade vuepress-theme-hope@2.0.0-beta.111`

Here is the repo to replicate the issue: https://github.com/JeremieLitzler/journal/tree/upgradeThemeToBeta111
Thanks again for the support provided.

**Comment by Mister-Hope** [comment, marked as answer]
Could you please all upgrade to latest version and provide a log?

This sounds related to #2326, however it seems to be related to upstream changes in vite, and I not pretty sure how it happenes, and in which situation I need to external `vuepress-shared` during ssr.

I believe the solution in the issue will help(even if you are using 101)

**Reply by JeremieLitzler** [reply]
I will try to upgrade to vuepress 2 beta 53 and your theme in beta 116

**Reply by JeremieLitzler** [reply]
@Mister-Hope, upgrading to latest versions did the trick. 

Thanks.
